.gitignore

```diff
@@ -85,7 +85,7 @@ ipython_config.py
 # pyenv
 #   For a library or package, you might want to ignore these files since the code is
 #   intended to run in multiple environments; otherwise, check them in:
-# .python-version
+.python-version
```

Comment:
I'm using `pyenv` myself too and I've found having this file checked in to the repo conflicting with my config, I believe this should live locally only. Probably same for replit, I don't use it myself but I guess it's a similar case.

Reply:
Honestly, i don't use `pyenv` in my workflow. I really don't know much about it, so adding `.python-version` to the `.gitignore` is ok for me.

To the `.replit` file:
In general, i don't think it's needed. But having it provides a better experience using this repository on [replit](https://replit.com/).
So please re-add the `.replit` file.